mandatory/3-function-output.js

```diff
@@ -1,9 +1,15 @@
 // Add comments to explain what this function does. You're meant to use Google!
+
+// Math.random() returns a random number between 0 (inclusive),  and 1 (exclusive):
+// For example: 0.40020936983011146
+// Math.random() always returns a number lower than 1.
 function getNumber() {
   return Math.random() * 10;
 } // The Math.random() function returns a floating-point, pseudo-random number in the range 0 to less than 1 (inclusive of 0, but not 1)with approximately uniform distribution over that range — which you can then scale to your desired range. so the getnumber function multiplies the psuedo random number between 0 and 10 however not 10.
 
 // Add comments to explain what this function does. You're meant to use Google!
+// The concat() method is used to join two or more arrays.
+This method does not change the existing arrays, but returns a new array, containing the values of the joined arrays.
```

Comment:
Hi Douglas great work buddy, you forgot to add slashes to make line 12 a comment.

Reply:
Thank you Jontaz77 for your comment. I'll review it and make the necceassary changes.